model_cards/English/toxicity-tensorflowjs.md

```diff
@@ -0,0 +1,123 @@
+# Toxicity
+
+The toxicity model classifies whether a comment is a rude, disrespectful, or
+unreasonable comment that is likely to make people leave a discussion.
+
+## Overview
+
+A modestly sized model based upon the Universal Sentence Encoder lite, suitable
+for client side (in browser) classificiation applications.
+
+![](toxicity-tensorflowjs-auc.png)
+
+## Intended use
+
+#### Human-assisted moderation
+
```

Comment:
Instead of the intended uses and uses to avoid that are here, maybe just reference those in the other model card and here highlight that the intended use is in-browser privacy sensitive use cases, and highlight that if a local model is not needed, the API is likely to provide better results.

Reply:
Done.